bukkit/src/main/java/com/viaversion/viaversion/ViaVersionPlugin.java

```diff
@@ -75,6 +76,14 @@ public ViaVersionPlugin() {
         conf.reload();
     }
 
+    /**
+     * @deprecated use {@link Via#getAPI()} instead
+     */
+    @Deprecated(forRemoval = true)
+    public static ViaVersionPlugin getInstance() {
+        return instance;
+    }
```

Comment:
Don't move this

Reply:
In my memory I didn't do this, maybe my IDE did it automatically. 

Already rolled back the operation.